.eslintignore

```diff
@@ -1,6 +1,9 @@
-/dist
-/static/framework
+dist
 node_modules
 
 # this is only for prettier (atom extension doesn't support .prettierignore)
 package.json
+
+# Temporary build files of the style guides
+packages/styleguides/ec/static/scripts
```

Comment:
Why here and not in the sub `packages/styleguides/*/.gitignore` files?

Reply:
> Only one .eslintignore file can be used at a time, so .eslintignore files other than the one in the current working directory will not be used.

I tried, but it appeared the eslint doesn't work as gitignore: https://eslint.org/docs/2.0.0/user-guide/configuring#ignoring-files-and-directories

Reply:
I didn't catch it was `.eslintignore` and not `.gitignore` 😅 
You're right, there should only be one `.eslintignore` but gitignore's syntax is still valid (i.e. you could use `packages/styleguides/*/static/scripts`)

Reply:
Alright, I used this approach to save a line, thanks